GitSavvy.sublime-settings

```diff
@@ -103,6 +103,30 @@
      */
     "close_panel_for": [],
 
+    /*
+        Whether git and githook output is logged live
+        If false, results are shown after execution.
+    */
+    "live_panel_output": false,
+
+    /*
+        Blacklist commands you don't want in the output panel at all.
+        Always start each entry with git ...
+    */
+    "show_panel_blacklist": [
```

Comment:
it seems a bit fragile to blacklist the full commands. Btw, why do we need to blacklist these commands at the first place?

Reply:
Let's say I enable live panel for `status`. Without this blacklist, I would then show a new output every time I changed the file when trying to update the status bar. 

We could take another approach to this and rend one more argument to the `git`method. Then add `ignore_live_log` to those very uninteresting commands.

Reply:
is it even useful to allow live panel for `status`? I thought it is mainly useful for `git pull` and `git push`. It doesn't look like there are any hooks which are related to `git status`. 

Reading one more argument is also not a good solution, but I have no suggestions neither.

Reply:
I have enabled it for `apply` not because I there are any hooks to apply but because I want to see what is being applied. Right after the apply was done it would overwrite it with one of the commands with I blacklisted. At that time I might I enable live log for everything.